TestingApp/ViewController.swift

```diff
@@ -52,6 +52,25 @@ class ViewController: UIViewController {
             print("End engagement operation has been executed. Result='\(result)'.")
         }
     }
+
+    @IBAction private func hackoFileTapped() {
```

Comment:
do we need this or is it just for dev purposes?

Reply:
0b694b137f305fd20abe136c88cabe05bbd5371b